src/test/java/ts/parking/test/CashCalculationTests.java

```diff
@@ -0,0 +1,33 @@
+package ts.parking.test;
+
+import static org.junit.Assert.assertEquals;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.time.LocalDateTime;
+
+import org.junit.Test;
+
+import ts.parking.model.Currency;
+import ts.parking.model.DriverType;
+import ts.parking.service.CashCalculations;
+
+public class CashCalculationTests {
```

Comment:
Bardzo skomplikowana logika (policz ify/branche), a tylko dwa testy?

Reply:
Racja, powinno być więcej testów. Pozwoliłoby to np. uniknąć błędu z ifa dla wartości parkingu poniżej jednej godizny, wspomniany w komentarzu wczesniej.